wikia/common/mw_database/connection.py

```diff
@@ -28,10 +42,27 @@ def _query(self, query, *args, **kwargs):
             log_text += ' (with args: {})'.format(kwargs['args'])
         logger.debug(log_text)
         def do_exec_query(cursor):
+            time_started = time.time()
+
             returned = cursor.execute(query, *args, **kwargs)
             affected = cursor.rowcount
             all_rows = cursor.fetchall()
 
+            # log SQL - sampled at 1%
+            if random.random() < 0.01:
+                extra = {}
+                extra.update(self.connection_info.__dict__ or {})
+                extra.pop('password', None)
+                extra['affected'] = affected
```

Comment:
`self.connection_info` is a `ConnectionDetails` instance which was extended to carry information about `cluster` and `master` flag and is being logged already.